sunpy/coordinates/wcs_utils.py

```diff
@@ -24,17 +26,22 @@ def solar_wcs_frame_mapping(wcs):
 
     dateobs = wcs.wcs.dateobs or None
 
```

Comment:
Note to self: raise a warning here if `wcs` has `heliographic_observer` or `rsun` attributes.